src/background.js

```diff
@@ -280,23 +274,24 @@ var pb = {
   * * set up listener to recheck blocked domains and DNT domains
   */
   initializeDNT: function(){
-    pb.updateDNTPolicyHashes();
-    pb.recheckDNTPolicyForDomains();
-    setInterval(pb.recheckDNTPolicyForDomains, pb.utils.oneHour());
-    setInterval(pb.updateDNTPolicyHashes, pb.utils.oneDay() * 4);
+    this.updateDNTPolicyHashes();
+    this.recheckDNTPolicyForDomains();
+    setInterval(this.recheckDNTPolicyForDomains, this.utils.oneHour());
+    setInterval(this.updateDNTPolicyHashes, this.utils.oneDay() * 4);
   },
 
   /**
   * Fetch acceptable DNT policy hashes from the EFF server
   */
   updateDNTPolicyHashes: function(){
-    pb.utils.xhrRequest(pb.DNT_POLICIES_URL, function(err,response){
+    var thisStorage = this.storage
+    utils.xhrRequest(this.DNT_POLICIES_URL, function(err,response){
```

Comment:
Should the following two `this.DNT_POLICIES_URL`'s use the *constants* variable also?